db/convenience.cc

```diff
@@ -56,8 +58,8 @@ Status VerifySstFileChecksum(const Options& options,
   std::unique_ptr<RandomAccessFileReader> file_reader(
       new RandomAccessFileReader(
           std::move(file), file_path, ioptions.clock, nullptr /* io_tracer */,
-          nullptr /* stats */, 0 /* hist_type */, nullptr /* file_read_hist */,
-          ioptions.rate_limiter.get()));
+          nullptr /* stats */, Histograms::SST_READ_MICROS /* hist_type */,
```

Comment:
In the places you changed 0 -> Histograms::SST_READ_MICROS, should we also provide `stats`?

Reply:
Fixed